internal/controller/controllers/agent_controller.go

```diff
@@ -348,7 +348,7 @@ func (r *AgentReconciler) populateEventsURL(log logrus.FieldLogger, agent *aiv1b
 }
 
 func (r *AgentReconciler) eventsURL(log logrus.FieldLogger, clusterId, infraEnvId, agentId string) (string, error) {
-	eventsURL := fmt.Sprintf("%s%s/v1/clusters/%s/events?host_id=%s", r.ServiceBaseURL, restclient.DefaultBasePath, clusterId, agentId)
+	eventsURL := fmt.Sprintf("%s%s/v2/events?cluster_id=%s&host_id=%s", r.ServiceBaseURL, restclient.DefaultBasePath, clusterId, agentId)
```

Comment:
Why does agent controller have a cluster id parameter? 
With kube-api we can return all agent events, @avishayt what do you think?

Reply:
Yes, I agree

Reply:
Probably because in V1, you had to provide the cluster_id as a part of the endpoint (and host_id was optional).
Now with V2, we have the option to query only with host_id. Probably it makes more sense.

I'll wait for @avishayt's response before I change it.

Reply:
I already responded :)